src/App/Fossa/VSI/DynLinked/Util.hs

```diff
@@ -16,6 +18,10 @@ import System.Info qualified as SysInfo
 hasSetUID :: Monad m => Path t File -> m Bool
 hasSetUID _ = pure False
 
+-- | The root of the primary file system.
+fsRoot :: Path Abs Dir
+fsRoot = $(mkAbsDir "C:/")
```

Comment:
IIRC it's not safe to assume `C:\` is the system root on windows, but I'm not going to block on this edge case

Reply:
I don't know of a way to change the primary system root from `C:\`, so this is likely a very far out edge case.  We can let someone run into this and then figure out a fix from there.